packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -5979,6 +5983,27 @@ export default class Meeting extends StatelessWebexPlugin {
     return !this.isRoapInProgress;
   }
 
+  /**
+   *  media failed, so collect a stats report from webrtc using the wcme connection to grab the rtc stats report
+   * send a webrtc telemetry dump to the configured server using the internal media core check metrics configured callback
+   * @param {String} callFrom - the function calling this function, optional.
+   * @returns {Promise<void>}
+   */
+  private forceSendStatsReport = async ({callFrom}: {callFrom?: string}) => {
+    const LOG_HEADER = `Meeting:index#forceSendStatsReport --> called from ${callFrom} : `;
+    try {
+      await this.mediaProperties?.webrtcMediaConnection?.forceRtcMetricsSend();
+      LoggerProxy.logger.info(
+        `${LOG_HEADER} failed but successfully uploaded available webrtc telemetry statistics`
```

Comment:
minor thing, but this log says "failed" so it assumes that `forceSendStatsReport()` was called because of a failure, which is true right now, but it might be called for other reasons too, better to just say "successfully uploaded available webrtc telemetry statistics"

Reply:
fixed